### ISSUE TITLE
Peter Lowe's list now blocks adf.ly content

### ISSUE THREAD

**Issue by lewisje** [uBlock-LLC/uBlock#1374]
It's good to block the ads, but not to keep users from getting the content behind those links; the uBlock Unbreak list should counteract this. (You could also change the URL from which you get the list, to include the parameter `&skip[]=adf.ly` as I did for the PAC implementation.)

**Comment by Temptin** [comment]
NO IT SHOULDN'T. JESUS FUCKING CHRIST TAKE IT UP WITH THE BLOCKLIST MAINTAINERS WHO BROKE IT, NOT WITH THE AUTHORS OF A RANDOM ADDON. PEOPLE LIKE YOU IS WHY GORHILL LEFT THE PROJECT. Morons reporting issues with blocklists as if they had ANYTHING to do with uBlock...

**Comment by chrisaljoudi** [comment]
Hi @lewisje!

Hmm, uBlock's built-in filter list does include [an exception filter to counteract that](https://github.com/chrisaljoudi/uBlock/blob/master/assets/ublock/filters.txt#L76). From what I can tell by testing, this does its job well.

Am I missing something?

**Comment by Temptin** [comment]
@chrisaljoudi Nope, you are correct. I hunted the web for a test link and found http://adf.ly/bJlzi (not sure what it links to) and yep it all works. Using default uBlock list subscriptions, recently updated, and includes Peter Lowe of course.

@lewisje Even if this was an issue (which it isn't), it has NOTHING to do with uBlock. Take these issues up with Peter Lowe and stop burdening the bugtracker of an unrelated project (uBlock). You are the reason Gorhill left: constant 3rd party blocklist "issues". Why do people insist on posting blocklist issues on the project page of a blocker? Go to the source of the issue!

@chrisaljoudi Why not create a new tag called "3rd-party blocklist issue, wontfix" and tag + insta-close ALL blocklist issues like this? Gorhill didn't deserve the issue tracker spam of unrelated issues, and neither do you. Just close them and move on to actual uBlock issues.

**Comment by lewisje** [comment]
@Temptin I was actually one of the more helpful users before the split (and after), not one of the idiots who complains about issues that turned out to be related to a third-party list that _isn't_ counteracted by default in this extension.

I posted the issue _precisely because_ I knew from previous issues that the uBlock Unbreak list is indeed updated to fix those third-party lists that are loaded by default, and this particular issue with Peter Lowe's list is one I noticed (via the PAC file) a few minutes before I posted this issue, so I thought that entry had just been added to the list and the uBlock team would need a heads-up. I don't know when that counteracting filter was added to uBlock Unbreak, so for all I know it could have been prompted by this very issue (I had not seen an issue filed about adf.ly before).

TL;DR: This actually is the kind of issue that the uBlock team works on, and I'm not the typical idiot who clogs the support queue.

**EDIT** I found an old issue related to adf.ly, filed way before I even heard of uBlock, but I'm still not sure why I just recently saw it show up in the PAC file version; I mean that site wasn't breaking for me until the day I posted this issue, that issue was posted last July, and I was using Peter Lowe's PAC file the whole time: https://github.com/chrisaljoudi/uBlock/issues/63

**Comment by Temptin** [comment]
@lewisje Great to hear that you've done your homework. I'm just annoyed at the low quality of "bug" reports on this issue tracker, and feel the pain of the devs. Yours looked like another one of those. Sorry... ;-)

I'd prefer if the devs can focus on actual bug reports and not blocklist issues/feature ideas. That would allow them to work on the fun parts of the project and not act as unpaid support for something that isn't even their fault. It was the reason Gorhill left and if it's fixed it may even be enough to merge the project back together again... maybe...

I can see three solutions to the constant "site X is broken" and "awesome feature idea!!!" reports:
- 1. Host issues on another bugtracking/ticket solution which auto-scans the text and guesses what the ticket is about, to speed up processing. Seeing that something is probably just a blocklist problem or feature idea ahead of time lets the dev skim through the text without having to read it as carefully. It would also offer "quick-response templates" where the devs can put some boilerplate text for use when closing such frivolous tickets.
- 2. Just relentlessly tag "3rd-party blocklist issue, wontfix" + close all blocklist related issues without wasting time even commenting. Do the same with "my amazing feature idea" type threads. And all the "I demand that you add hotkey support because I want that feature even though I haven't paid you anything for your time" garbage.
- 3. Do what AdBlock Plus does: Go commercial and hire paid staff that constantly responds to users problems with blocklists (their forums are a nightmare). This one is just a pipe dream, though.

I'm just worried because it seems unlikely that @chrisaljoudi can keep this project up on his own without burnout since he's just 17 and has to think about life and school (though he's a skilled coder for sure!), and @gorhill seems to have found happiness working on his much quieter fork (much smaller issue tracker). Since gorhill has demonstrated that he'd be happy to host and work on the Chrome version and to even run his own issue tracker, why not work out the issue tracker problems on the main repo and merge again? It'd be best for uBlock if they merge, and just undo the minor "per site switches" removal that chris did, to get the projects back in sync...

Seems that what's really needed here is more relentless closing of tickets, and less engagement with the _lazy_ users. It was crazy back then when gorhill would reply to every user issue in detail, both here on the tracker and on reddit and on various forums. I can see why he burned out...

The ball is in @gorhill 's court, since Chris has repeatedly offered to hand the repo back. For the best future for uBlock I hope they reach an agreement someday. ;-)
